qcodes/logger/logger.py

```diff
@@ -18,9 +18,15 @@
 
 from typing import Optional, Union, Sequence
 
+from applicationinsights.logging.LoggingHandler import LoggingHandler
+
 import qcodes as qc
 import qcodes.utils.installation_info as ii
 
+# We need to declare the type of this global variable up here. See
+# https://github.com/python/mypy/issues/5732 for reference
+telemetry_handler: LoggingHandler
```

Comment:
Is it worth considering wrapping this in "if typechecking" and move the imports down to make appinsights an optional dependency?